go.mod

```diff
@@ -4,21 +4,14 @@ go 1.11
 
 require (
 	github.com/campoy/embedmd v1.0.0 // indirect
+	github.com/go-kit/log v0.2.0
 	github.com/goburrow/modbus v0.0.0-20161010020032-f7afd8db7d8d
 	github.com/goburrow/serial v0.0.0-20170301104454-d490ecc9d6a1 // indirect
-	github.com/golang/protobuf v1.3.1 // indirect
-	github.com/hashicorp/errwrap v0.0.0-20141028054710-7554cd9344ce // indirect
-	github.com/hashicorp/go-multierror v0.0.0-20161216184304-ed905158d874
-	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
-	github.com/kr/pretty v0.1.0 // indirect
-	github.com/prometheus/client_golang v0.9.2
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90 // indirect
-	github.com/prometheus/common v0.3.0
-	github.com/prometheus/procfs v0.0.0-20190412120340-e22ddced7142 // indirect
-	github.com/sirupsen/logrus v1.4.1 // indirect
-	github.com/stretchr/testify v1.3.0 // indirect
+	github.com/hashicorp/go-multierror v1.0.0
+	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/common v0.15.0
+	github.com/prometheus/exporter-toolkit v0.5.0
```

Comment:
This is a very old version, you should probably update this to the latest. We have simplified the interface and flags. See https://github.com/prometheus/node_exporter/pull/2393.

Reply:
`--telemetry-listen-address` will be replaced by the exporter toolkit `--web.listen-address`. The `--modbus-listen-address` will stay the same. But should probably be renamed to match the usual Prometheus conventions.